python/lsst/sims/featureScheduler/features/conditions.py

```diff
@@ -0,0 +1,302 @@
+            approximate equation for converting RA,Dec to alt,az.
+        az : np.array
+            Azimuth of each healpixel (radians). Recaclulated if mjd is updated. Uses fast
+            approximate equation for converting RA,Dec to alt,az.
+        pa : np.array
+            The parallactic angle of each healpixel (radians). Recaclulated if mjd is updated.
+            Based on the fast approximate alt,az values.
+        clouds : float
+            The fraction of sky covered by clouds. (In the future might update to transparency map)
+        slewtime : np.array
+            Healpix showing the slewtime to each healpixel center (seconds)
+        current_filter : str
+            The name of the current filter. (expect one of u, g, r, i, z, y).
+        mounted_filters : list of str
+            The filters that are currently mounted and thu available (expect 5 of u, g, r, i, z, y)
+        night : int
+            The current night number (days). Probably starts at 1.
+        lmst : float
+            The local mean sidearal time (hours). Updates is mjd is changed.
+        skybrightness : dict of np.array
+            Dictionary keyed by filtername. Values are healpix arrays with the sky brightness at each
+            healpix center (mag/acsec^2)
+        FWHMeff : dict of np.array
+            Dictionary keyed by filtername. Values are the effective seeing FWHM at each healpix
+            center (arcseconds)
+        M5Depth : dict of np.array
+            the 5-sigma limiting depth healpix maps, keyed by filtername (mags). Will be recalculated
+            if the skybrightness, seeing, or airmass are updated.
+        queue : list of observation objects
+            The current queue of observations waiting to be executed.
+        moonAlt : float
+            The altitude of the Moon (radians)
+        moonAz : float
+            The Azimuth of the moon (radians)
+        moonRA : float
+            RA of the moon (radians)
+        moonDec : float
+            Declination of the moon (radians)
+        moonPhase : float
+            The Phase of the moon. (fraction, 0=new moon, 1=full moon)
+        sunAlt : float
+            The altitude of the sun (radians).
+        sunAz : float
+            The Azimuth of the sun (radians).
+        last_twilight_end : float
+            The MJD when the last twilight ended. (days)
+        next_twilight_start : float
+            The MJD when the next twilight will start. (days) XXX--need to look up how this twilight is
+            defined! For speedObservatory it's sunAlt of -18 degrees
+        telRA : float
+            The current telescope RA pointing (radians).
+        telDec : float
+            The current telescope Declination
+        cloud_map : np.array
+            A healpix map with the cloud coverage. XXX-expand, is this bool map? Transparency map?
+        HA : np.array
+            Healpix map of the hour angle of each healpixel (radians).
+
```

Comment:
Good idea! The airmass map historically comes in from the telemetry stream because it was a pre-computed value. Might make sense to move that calc into here.